service/frontend/workflow_handler.go

```diff
@@ -4815,9 +4815,11 @@ func (wh *WorkflowHandler) ListBatchOperations(
 		Namespace:     request.GetNamespace(),
 		PageSize:      request.PageSize,
 		NextPageToken: request.GetNextPageToken(),
-		Query: fmt.Sprintf("%s = '%s' and %s = '%s'",
+		Query: fmt.Sprintf("(%s = '%s' or %s = '%s') and %s = '%s'",
```

Comment:
Lets just remove the workflow type filter and make sure this is merged ASAP because what we have in main is broken.